README.Rmd

```diff
@@ -12,12 +12,11 @@ knitr::opts_chunk$set(
   fig.path = "tools/readme/README-"
 )
 ```
+
 # ggeasy <img src='man/figures/logo.png' align="right" height="138.5" />
-[![Covrpage Summary](https://img.shields.io/badge/covrpage-Last_Build_2019_12_04-yellowgreen.svg)](https://github.com/jonocarroll/ggeasy/tree/cleanup/sept18/tests/README.md)
+[![Covrpage Summary](https://img.shields.io/badge/covrpage-Last_Build_2019_12_05-yellowgreen.svg)](http://tinyurl.com/rh6otye)
```

Comment:
Any reason for the shortURLs?

Reply:
Cran got fussy with the link in the readme for some reason.